src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -242,6 +245,30 @@ def is_running(self) -> bool:
             cluster_name=self._k3d_cluster_name
         )
 
+    def list_manual_setup_steps(self) -> None:
+        """Logs manual steps needed to setup the Kubeflow local orchestrator."""
+        global_config_dir_path = zenml.io.utils.get_global_config_directory()
+        logger.error("Unable to spin up local Kubeflow Pipelines deployment.")
+        logger.info(
+            "If you wish to spin up this Kubeflow local orchestrator manually, "
+            "please enter the following commands (substituting where appropriate):\n"
+        )
+        logger.info(
```

Comment:
The problem with this is that the `registry.yaml` file has to be exactly the one we write in order KFP to be able to pull the images. So I think we should write the yaml file even in case of failure and include the path in the command (which would also require a fixed REGISTRY_NAME).